AppDB/appscale/datastore/cassandra_env/cassandra_interface.py

```diff
@@ -1023,8 +1080,8 @@ def delete_entities_tx(self, app, txid, entity_keys):
     insert = self.session.prepare("""
       INSERT INTO transactions (txid_hash, operation, namespace, path, entity)
       VALUES (?, ?, ?, ?, ?)
-      USING TTL 120
-    """)
+      USING TTL {ttl}
+    """.format(ttl=dbconstants.MAX_TX_DURATION * 2))
```

Comment:
It just gives some reasonable padding to ensure the state is present for the duration of the transaction. Eventually, we may need it to exist a bit longer than the transaction for the taskqueue to query the status of a transaction.